examples/org.eclipse.swt.snippets/src/org/eclipse/swt/snippets/Snippet282.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2016 IBM Corporation and others.
+ * Copyright (c) 2022 IBM Corporation and others.
```

Comment:
This is wrong change of Copyright - It should  become 2016, 2022

Reply:
Made this change.